packages/pharos/src/components/switch/pharos-switch.scss

```diff
@@ -0,0 +1,122 @@
+@use '../../utils/scss/mixins';
+
+.input-wrapper {
+  @include mixins.option-wrapper;
+
+  align-items: center;
+  column-gap: var(--pharos-spacing-1-x);
+}
+
+.switch__control {
+  height: 32px;
+  width: 70px;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  cursor: pointer;
+  border: 1px solid;
+  border-color: var(--pharos-color-marble-gray-80);
+  border-radius: 2rem;
+  background-color: var(--pharos-color-marble-gray-94);
+  transition:
+    background-color 0.3s ease-out,
```

Comment:
I do like the way this works, but see it is not one of our [transition tokens](https://pharos.jstor.org/design-tokens/transitions). I wonder if switching to `--pharos-transition-duration-default` which is 250 ms so it matched or other transitions would be make a noticeable difference?(cc @fswlee as this is mostly a design decision)

Reply:
FWIW 50ms is generally imperceptible to humans so I'd go for the token value too 😄

Reply:
✅